Makefile

```diff
@@ -505,12 +513,11 @@ eks-a-e2e:
 			make eks-a-release GIT_VERSION=$(DEV_GIT_VERSION); \
 			scripts/get_bundle.sh; \
 		else \
-			make check-eksa-components-override; \
```

Comment:
never going to be an override here?

Reply:
That override was solely for the eks-a components not being in S3 at the time of running e2e tests. I figured that now since we always generate the bundle before running tests, we should have it in S3 so won't need overrides.